shelley/chain-and-ledger/formal-spec/update.tex

```diff
@@ -182,7 +218,7 @@ \subsection{Protocol Parameter Update Proposals}
       & \fun{votedValue_T} \in (\KeyHashGen\mapsto\type{T}) \to \type{T}^?\\
       & \fun{votedValue_T}~\var{vs} =
         \begin{cases}
-          t & \exists t\in\range{vs}~(|vs\restrictrange t|\geq 5) \\
+          t & \exists t\in\range{vs}~(|vs\restrictrange t|\geq \Quorum) \\
```

Comment:
we should probably also replace the hard coded value in the `UTXOW` rule for the instantaneous rewards check